openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TableRepository.java

```diff
@@ -51,25 +51,8 @@
 import org.openmetadata.schema.entity.data.Table;
 import org.openmetadata.schema.tests.CustomMetric;
 import org.openmetadata.schema.tests.TestSuite;
-import org.openmetadata.schema.type.Column;
-import org.openmetadata.schema.type.ColumnJoin;
-import org.openmetadata.schema.type.ColumnProfile;
-import org.openmetadata.schema.type.ColumnProfilerConfig;
-import org.openmetadata.schema.type.DailyCount;
-import org.openmetadata.schema.type.DataModel;
-import org.openmetadata.schema.type.EntityReference;
-import org.openmetadata.schema.type.Include;
-import org.openmetadata.schema.type.JoinedWith;
-import org.openmetadata.schema.type.Relationship;
-import org.openmetadata.schema.type.SystemProfile;
-import org.openmetadata.schema.type.TableConstraint;
-import org.openmetadata.schema.type.TableData;
-import org.openmetadata.schema.type.TableJoins;
-import org.openmetadata.schema.type.TableProfile;
-import org.openmetadata.schema.type.TableProfilerConfig;
-import org.openmetadata.schema.type.TagLabel;
-import org.openmetadata.schema.type.TaskDetails;
-import org.openmetadata.schema.type.TaskType;
+import org.openmetadata.schema.type.*;
```

Comment:
we can disable intellij from importing * 

https://docs.open-metadata.org/v1.1.2/developers/contribute/build-code-and-run-tests/openmetadata-server#:~:text=Configure%20IntelliJ%20to%20disable%20the%20%5Bwild%2Dcard%20imports%5D%20(https%3A//www.jetbrains.com/help/idea/creating%2Dand%2Doptimizing%2Dimports.html%23disable%2Dwildcard%2Dimports)